### PR TITLE
Complete window.localStorage and OutfitList Components

### PR DESCRIPTION
**Pull request by VesLan**

### CODE REVIEW

File: client/src/components/related/outfit-components/OutfitProduct.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  return (
+    <div>
+      <div className="outfit-item">
+        <MdOutlineDeleteOutline className="outfit-delete-icon" onClick={handleDelete} />
+        <img
+          id="outfit-img"
+          src={outfitStyle.photos[0].thumbnail_url}
+          alt={outfitProd.description}
+        // onMouseEnter={() => setShowImg(true)}
```
Comment (eriknewland) [inline client/src/components/related/outfit-components/OutfitProduct.jsx:51]: Couple of lingering comments to delete

File: client/src/components/related/related-components/RelatedList.jsx [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
   };
 
   const scrollRight = () => {
-    posIndex < end ? setPosIndex(posIndex + 1) : null;
+    posIndex < endOfRelatedList ? setPosIndex(posIndex + 1) : null;
     ref.current.scrollLeft += 15 * 16;
   };
 
+  useEffect(() => {
+
```
Comment (eriknewland) [inline client/src/components/related/related-components/RelatedList.jsx:24]: Is this useEffect doing anything?

**Review by eriknewland — CHANGES_REQUESTED** [review]
Overall looks very good, a couple lingering comments and a stray useEffect that is not doing anything. I want to go over "eslint-disable no-unused-expressions" before we merge, and also would like to see a short demo of the second carousel. Lastly, Bolton realized our linter wasn't utilizing ECMA9 which may have been causing strange lint errors.

File: client/src/components/related/related-components/RelatedList.jsx (hunk 2 above)
Comment (VesLan) [reply client/src/components/related/related-components/RelatedList.jsx:24]: Oh no I move it somewhere else, just delete it

File: client/src/components/related/outfit-components/OutfitProduct.jsx (hunk 1 above)
Comment (VesLan) [reply client/src/components/related/outfit-components/OutfitProduct.jsx:51]: Ok.

**Review by eriknewland — APPROVED** [review]
Thanks for the demo and making those small clean-ups!

**Pull request merged**